gradle.properties

```diff
@@ -1,7 +1,3 @@
-# https://kotlinlang.org/docs/mpp-share-on-platforms.html
-kotlin.mpp.enableGranularSourceSetsMetadata=true
-kotlin.native.enableDependencyPropagation=false
```

Comment:
These removed options are deprecated in Kotlin 1.6.20 per [release announcement](https://kotlinlang.org/docs/whatsnew1620.html#hierarchical-structure-support-for-multiplatform-projects).